LibraryApp/src/components/shared/image-picker.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Pressable,
+  Image,
+  PermissionsAndroid,
+} from 'react-native';
+import {Label} from 'components/shared/form-controls';
+import {Colors, Global, Spacing, Typography} from 'stylesheets';
+import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
+import {cameraBlack, imageBlack} from 'constants/icons';
+
+const {
+  rowFlex,
+  flexEnd,
+  columnFlex,
+  spaceBetween,
+  verticalCenter,
+  borderWidth1,
+} = Global;
+const {iconFont, fs20} = Typography;
+const {px10, mb20} = Spacing;
+const {darkGrey, white} = Colors;
+
+class ImagePicker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      imagePath: {},
+    };
+  }
+
+  // ------- request Camera Permission ----------- //
+  requestCameraPermission = async () => {
+    if (Platform.OS === 'android') {
+      try {
+        const granted = await PermissionsAndroid.request(
+          PermissionsAndroid.PERMISSIONS.CAMERA,
+          {
+            title: 'Camera Permission',
+            message: 'App needs camera permission',
+          },
+        );
+        // ----- If CAMERA Permission is granted ------- //
+        return granted === PermissionsAndroid.RESULTS.GRANTED;
+      } catch (err) {
+        console.warn(err);
+        return false;
+      }
+    } else return true;
+  };
+
+  // ------------ request Write Permission to Store Captured Image to Device --------- //
+  requestExternalWritePermission = async () => {
+    if (Platform.OS === 'android') {
```

Comment:
what about iOS permission? can we manage that as well?

Reply:
Added ios permissions as well.